spring-batch-core/src/main/java/org/springframework/batch/core/job/AbstractJob.java

```diff
@@ -297,7 +302,11 @@ public final void execute(JobExecution execution) {
 		}
 
 		JobSynchronizationManager.register(execution);
-
+		LongTaskTimer longTaskTimer = LongTaskTimer.builder(METRICS_PREFIX + "jobs.active")
+				.description("Active jobs")
```

Comment:
I agree. I changed `jobs.active` to `job.active` for consistency. However, the description should remain in plural because this is what it really is: Currently "in flight" jobs to use micrometer terminology.